lib/workload/components/python-lambda-layer/index.ts

```diff
@@ -0,0 +1,41 @@
+import { Construct } from 'constructs';
+import { PythonLayerVersion } from '@aws-cdk/aws-lambda-python-alpha';
+import * as lambda from 'aws-cdk-lib/aws-lambda';
+
+export interface PythonLambdaLayerConstructProps {
+  layer_name: string;
+  layer_directory: string;
+  layer_description: string;
+}
+
+export class PythonLambdaLayerConstruct extends Construct {
+  public readonly lambda_layer_arn: string;
+  public readonly lambda_layer_version_obj: PythonLayerVersion;
+
+  constructor(scope: Construct, id: string, props: PythonLambdaLayerConstructProps) {
+    super(scope, id);
+
+    this.lambda_layer_version_obj = new PythonLayerVersion(this, 'python_lambda_layer', {
```

Comment:
@alexiswl Just minor request, pls.

I just notice these as post-mortem review. When you have spare time down the track, could you please follow up refactoring naming convention on these CDK variables. There seems to be mixed use of `snake_case` and `camelCase`.

ditto - current guidance on TypeScript
https://github.com/umccr/orcabus/blob/main/README.md#typography

This apply to both `bsRunsUploadManager` and `ICAv2CopyBatchManager` services that have merged.

Reply:
And.. separate PRs for these refactor, pls.

Reply:
> And.. separate PRs for these refactor, pls.

Ah that's going to cause some annoying merge conflicts, the PytohnLayer construct example above will rename the attribute lambda_layer_version_obj to lambdaLayerVersionObj, but the property of the bs runs upload manager stack (lambda_layer_obj will be renamed to lambdaLayerObj),  

so the layers attribute in any of the python functions will go from `layers: [props.lambda_layer_obj.lambda_layer_version_obj]` to `layers: [props.lambdaLayerObj.lambdaLayerVersionObj]` where `lambdaLayerObj` comes from one branch and `lambdaLayerVersionObj` comes from another branch.  

I think this would be easier to all be in one PR

Reply:
I mean, ok to be all in one go, refactor PR, yes.